app/client/styles/components/_Welcome.scss

```diff
@@ -2,6 +2,26 @@
 // This file contains all styles related to the Welcome component.
 // -----------------------------------------------------------------------------
 
+.splash {
+	width: 100%;
+	height: 100%;
+	z-index: -1;
+	position: absolute;
+}
+.serif {
+	font-family: $serif;
+}
+h1 {
+	margin: -20px 0px 30px 0px;
+	-webkit-text-stroke: 1.25px #1B1E1F;
+	font-weight: 800;
+	font-size: 2.25em;
+}
+
+input[type="button"] {
+	background-color: #D6D3CD;
+	color: #1B1E1F;
```

Comment:
Can these be sass variables? You reuse the colors a few times in this sheet, so we could abstract them at the top: $button-background-color: #D6D3CD